### PR TITLE
Specify rx version and match other dependencies in setup.py

### PR DESCRIPTION
**Pull request by kinow**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by kinow** [comment]
The change in version ranges is due to warnings such as

```
WARNING conda.models.version:get_matcher(528): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.*, but conda is ignoring the .* and treating it as 3
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
WARNING conda.models.version:get_matcher(528): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2

```

See comment in cylc-flow feedstock PR for more.

**Comment by kinow** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,11 +28,12 @@ requirements:
   run:
     - python
     - cylc-flow ==8.0a2
-    - jupyterhub >=1.1
-    - tornado >=6.0
-    - graphene-tornado >=2.6
+    - jupyterhub ==1.1.*
```
Comment (oliver-sanders) [inline recipe/meta.yaml:31]: Uses glob syntax which doesn't work in Conda?

Comment (kinow) [reply recipe/meta.yaml:31]: Hm? Did I forget to push a change here? I thought I had updated both recipes :thinking:

Comment (kinow) [reply recipe/meta.yaml:31]: Ha, forgot to push, sorry @oliver-sanders 

image

That's what happens when you finish PR's while trying to start cooking at the same time. Sorry!

**Pull request closed without merging**